e2e/app/valupdates.go

```diff
@@ -43,18 +43,35 @@ func FundValidatorsForTesting(ctx context.Context, def Definition) error {
 	}
 
 	// Iterate over all nodes, since all maybe become validators.
+	var eg errgroup.Group
 	for _, node := range def.Testnet.Nodes {
-		addr, _ := k1util.PubKeyToAddress(node.PrivvalKey.PubKey())
-		tx, _, err := fundBackend.Send(ctx, funder, txmgr.TxCandidate{
-			To:       &addr,
-			GasLimit: 100_000,
-			Value:    math.NewInt(1000).MulRaw(params.Ether).BigInt(),
+		eg.Go(func() error {
+			addr, _ := k1util.PubKeyToAddress(node.PrivvalKey.PubKey())
```

Comment:
Are we allowed to silence errors? Should we use a `Must*` method instead?

Reply:
thanks will fix